applications/maplab-server-node/src/maplab-server-ros-node.cc

```diff
@@ -22,35 +22,38 @@
 #include "maplab-server-node/maplab-server-node.h"
 
 DEFINE_string(
-    server_node_config_file, "",
+    maplab_server_node_config_file, "",
     "Path the the config YAML file for the maplab server node.");
 
 DEFINE_int32(
-    map_update_notification_subscriber_queue_size, 100,
-    "Size of ROS subscriber.");
+    maplab_server_map_update_topic_queue_size, 100, "Size of ROS subscriber.");
+
+DEFINE_string(
+    maplab_server_map_update_topic, "map_update_notification",
+    "Topic on which the map update notification message is received, it "
+    "contains the robot name and the map folder of the new map update.");
 
 namespace maplab {
 
-MaplabServerRosNode::MaplabServerRosNode(
-    const MaplabServerRosNodeConfig& config)
-    : config_(config), maplab_spinner_(common::getNumHardwareThreads()) {
+MaplabServerRosNode::MaplabServerRosNode(const MaplabServerNodeConfig& config)
```

Comment:
This function seems to be useless. I couldn't find it anywhere being called and it seems deprecated since it doesn't implement a lot of the behavior of the below function.

Reply:
it is used in the unit tests, where we start the node without setting up the ros callbacks.